AMW_business/src/main/resources/liquibase/changesets/base/data/initial.data.xml

```diff
@@ -18,25 +18,16 @@
         </sql>
     </changeSet>
 
-    <changeSet author="initial-data" id="initial-data-base-release-1-pg" dbms="postgresql">
```

Comment:
is not going to work, the boolean for MAINRELEASE needs to be an integer for oracle